diff -Nur -x '*.orig' -x '*.rej' e-smith-horde-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/registry.php/00header mezzanine_patched_e-smith-horde-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/registry.php/00header
--- e-smith-horde-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/registry.php/00header  2008-03-16 21:26:38.000000000 -0500
+++ mezzanine_patched_e-smith-horde-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/registry.php/00header        2008
-03-16 21:20:57.000000000 -0500
@@ -2,7 +2,7 @@
 /**
  * registry.php -- Horde application registry.
  *
- * $Horde: horde/config/registry.php.dist,v 1.255.2.20 2007/03/29 23:02:48 jan Exp $
+ * $Horde: horde/config/registry.php.dist,v 1.255.2.21 2007/10/11 00:28:54 jan Exp $
  *
  * This configuration file is used by Horde to determine which Horde
  * applications are installed and where, as well as how they interact.
diff -Nur -x '*.orig' -x '*.rej' e-smith-horde-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/registry.php/800WebRoo
t mezzanine_patched_e-smith-horde-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/registry.php/800WebRoot
--- e-smith-horde-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/registry.php/800WebRoot        2008-03-16 21:26:38.
000000000 -0500
+++ mezzanine_patched_e-smith-horde-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/registry.php/800WebRoot      2008
-03-16 21:23:37.000000000 -0500
@@ -9,7 +9,11 @@
             $_SERVER['SCRIPT_URL'];
         $hordedir = str_replace(DIRECTORY_SEPARATOR, '/', __FILE__);
         $hordedir = basename(preg_replace(';/config/registry.php$;', '', $hordedir));
-        $webroot = preg_replace(';/' . $hordedir . '.*;', '/' . $hordedir, $path);
+        if (preg_match(';/' . $hordedir . ';', $path)) \{
+            $webroot = preg_replace(';/' . $hordedir . '.*;', '/' . $hordedir, $path);
+        \} else \{
+            $webroot = '';
+        \}
     \} elseif (isset($_SERVER['PHP_SELF'])) \{
         $webroot = preg_split(';/;', $_SERVER['PHP_SELF'], 2, PREG_SPLIT_NO_EMPTY);
         $webroot = strstr(dirname(__FILE__), DIRECTORY_SEPARATOR . array_shift($webroot));

