diff -Nur -x '*.orig' -x '*.rej' e-smith-horde-1.13.0/root/home/httpd/html/horde/scripts/db/mysql_create_tables.sql mezzanine_patched_e-smith-horde-1.13.0/root/home/httpd/html/horde/scripts/db/mysql_create_tables.sql
--- e-smith-horde-1.13.0/root/home/httpd/html/horde/scripts/db/mysql_create_tables.sql	2003-11-24 11:50:39.000000000 -0600
+++ mezzanine_patched_e-smith-horde-1.13.0/root/home/httpd/html/horde/scripts/db/mysql_create_tables.sql	2006-10-01 21:18:43.000000000 -0500
@@ -3,19 +3,23 @@
 CONNECT horde;
 
 CREATE TABLE IF NOT EXISTS horde_users (
-    user_uid       varchar(255) not null,
-    user_pass      varchar(32) not null,
-    primary key (user_uid)
+    user_uid                    VARCHAR(255) NOT NULL,
+    user_pass                   VARCHAR(255) NOT NULL,
+    user_soft_expiration_date   INT,
+    user_hard_expiration_date   INT,
+    
+    PRIMARY KEY (user_uid)
 );
 
 GRANT SELECT, INSERT, UPDATE, DELETE ON horde_users TO horde@localhost;
 
 CREATE TABLE IF NOT EXISTS horde_prefs (
-    pref_uid        char(255) not null,
-    pref_scope      char(16) not null default '',
-    pref_name       char(32) not null,
-    pref_value      text null,
-    primary key (pref_uid, pref_scope, pref_name)
+    pref_uid        VARCHAR(200) NOT NULL,
+    pref_scope      VARCHAR(16) NOT NULL DEFAULT '',
+    pref_name       VARCHAR(32) NOT NULL,
+    pref_value      LONGTEXT NULL,
+    
+    PRIMARY KEY (pref_uid, pref_scope, pref_name)
 );
 
 GRANT SELECT, INSERT, UPDATE, DELETE ON horde_prefs TO horde@localhost;
