diff -Nur -x '*.orig' -x '*.rej' e-smith-horde-1.13.0/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/85HordeAccess mezzanine_patched_e-smith-horde-1.13.0/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/85HordeAccess
--- e-smith-horde-1.13.0/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/85HordeAccess	2006-03-15 17:51:23.000000000 -0600
+++ mezzanine_patched_e-smith-horde-1.13.0/root/etc/e-smith/templates/etc/httpd/conf/httpd.conf/85HordeAccess	2006-12-05 21:44:38.000000000 -0600
@@ -54,6 +54,11 @@
     deny from all
 </Directory>
 
+<Directory /home/httpd/html/horde/scripts>
+    order deny,allow
+    deny from all 
+</Directory>
+
 );
     }
     else
