diff -Nur -x '*.orig' -x '*.rej' e-smith-horde-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/conf.php/120AdminAuthentication mezzanine_patched_e-smith-horde-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/conf.php/120AdminAuthentication
--- e-smith-horde-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/conf.php/120AdminAuthentication	2006-12-24 02:38:21.000000000 -0600
+++ mezzanine_patched_e-smith-horde-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/conf.php/120AdminAuthentication	2006-12-24 02:32:40.000000000 -0600
@@ -1,2 +1,5 @@
 // 120AdminAuthentication
-$conf['auth']['admins'] = array('admin@{$DomainName}');
+{
+my $HordeAdmin = $horde{'Administration'} || 'disabled'; return "\$conf[\'auth\'][\'admins\'] = array('admin\@$DomainName')\;" unless ($HordeAdmin eq "disabled");
+$OUT = '$conf[\'auth\'][\'admins\'] = array(\'\');'
+}
