diff -Nur -x '*.orig' -x '*.rej' e-smith-horde-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/registry.php/120AppRegistryImp mezzanine_patched_e-smith-horde-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/registry.php/120AppRegistryImp
--- e-smith-horde-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/registry.php/120AppRegistryImp	2006-03-15 17:51:24.000000000 -0600
+++ mezzanine_patched_e-smith-horde-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/registry.php/120AppRegistryImp	1969-12-31 18:00:00.000000000 -0600
@@ -1,9 +0,0 @@
-// 120AppRegistryImp
-$this->applications['imp'] = array(
-    'fileroot' => dirname(__FILE__) . '/../imp',
-    'webroot' => $this->applications['horde']['webroot'] . '/imp',
-    'name' => _("Mail"),
-    'status' => 'active',
-    'provides' => 'mail',
-);
-
diff -Nur -x '*.orig' -x '*.rej' e-smith-horde-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/registry.php/130AppRegistryIngo mezzanine_patched_e-smith-horde-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/registry.php/130AppRegistryIngo
--- e-smith-horde-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/registry.php/130AppRegistryIngo	2006-03-15 17:51:23.000000000 -0600
+++ mezzanine_patched_e-smith-horde-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/registry.php/130AppRegistryIngo	1969-12-31 18:00:00.000000000 -0600
@@ -1,10 +0,0 @@
-// 130AppRegistryIngo
-$this->applications['ingo'] = array(
-    'fileroot' => dirname(__FILE__) . '/../ingo',
-    'webroot' => $this->applications['horde']['webroot'] . '/ingo',
-    'name' => _("Filters"),
-    'status' => 'active',
-    'provides' => array('mail/blacklistFrom', 'mail/showBlacklist', 'mail/whitelistFrom', 'mail/showWhitelist', 'mail/applyFilters', 'mail/canApplyFilters', 'mail/showFilters'),
-    'menu_parent' => 'imp'
-);
-
diff -Nur -x '*.orig' -x '*.rej' e-smith-horde-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/registry.php/170AppRegistryImpFolders mezzanine_patched_e-smith-horde-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/registry.php/170AppRegistryImpFolders
--- e-smith-horde-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/registry.php/170AppRegistryImpFolders	2006-03-15 17:51:24.000000000 -0600
+++ mezzanine_patched_e-smith-horde-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/registry.php/170AppRegistryImpFolders	1969-12-31 18:00:00.000000000 -0600
@@ -1,8 +0,0 @@
-// 170AppRegistryImp
-$this->applications['imp-folders'] = array(
-    'status' => 'block',
-    'app' => 'imp',
-    'blockname' => 'tree_folders',
-    'menu_parent' => 'imp',
-);
-
diff -Nur -x '*.orig' -x '*.rej' e-smith-horde-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/registry.php/210AppRegistryTurba mezzanine_patched_e-smith-horde-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/registry.php/210AppRegistryTurba
--- e-smith-horde-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/registry.php/210AppRegistryTurba	2006-10-05 21:44:12.000000000 -0500
+++ mezzanine_patched_e-smith-horde-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/registry.php/210AppRegistryTurba	1969-12-31 18:00:00.000000000 -0600
@@ -1,17 +0,0 @@
-// 210AppRegistryTurba
-$this->applications['turba'] = array(
-    'fileroot' => dirname(__FILE__) . '/../turba',
-    'webroot' => $this->applications['horde']['webroot'] . '/turba',
-    'name' => _("Address Book"),
-    'status' => 'active',
-    'provides' => array('contacts', 'clients'),
-    'menu_parent' => 'organizing'
-);
-
-$this->applications['turba-menu'] = array(
-    'status' => 'block',
-    'app' => 'turba',
-    'blockname' => 'tree_menu',
-    'menu_parent' => 'turba',
-);
-
