diff -Nur -x '*.orig' -x '*.rej' e-smith-horde-1.13.0/createlinks mezzanine_patched_e-smith-horde-1.13.0/createlinks
--- e-smith-horde-1.13.0/createlinks	2007-01-20 14:57:44.000000000 -0700
+++ mezzanine_patched_e-smith-horde-1.13.0/createlinks	2007-01-20 14:56:06.000000000 -0700
@@ -50,7 +50,7 @@
     50horde-2.2_to_3.0
     55horde-3.0_to_3.1
     56horde-3.1_alter_table
-    77mysql_update_privs
+    77horde_mysql_update_privs
     ))
 {
     templates2events("/etc/e-smith/sql/init/$_", $event);
@@ -73,7 +73,7 @@
     50horde-2.2_to_3.0
     55horde-3.0_to_3.1
     56horde-3.1_alter_table
-    77mysql_update_privs
+    77horde_mysql_update_privs
     ))
 {
     templates2events("/etc/e-smith/sql/init/$_", $event);
diff -Nur -x '*.orig' -x '*.rej' e-smith-horde-1.13.0/root/etc/e-smith/templates/etc/e-smith/sql/init/77horde_mysql_update_privs mezzanine_patched_e-smith-horde-1.13.0/root/etc/e-smith/templates/etc/e-smith/sql/init/77horde_mysql_update_privs
--- e-smith-horde-1.13.0/root/etc/e-smith/templates/etc/e-smith/sql/init/77horde_mysql_update_privs	1969-12-31 17:00:00.000000000 -0700
+++ mezzanine_patched_e-smith-horde-1.13.0/root/etc/e-smith/templates/etc/e-smith/sql/init/77horde_mysql_update_privs	2007-01-20 14:55:30.000000000 -0700
@@ -0,0 +1,3 @@
+#! /bin/sh
+
+exec mysql < /home/httpd/html/horde/scripts/db/mysql_update_privs.sql
diff -Nur -x '*.orig' -x '*.rej' e-smith-horde-1.13.0/root/etc/e-smith/templates/etc/e-smith/sql/init/77mysql_update_privs mezzanine_patched_e-smith-horde-1.13.0/root/etc/e-smith/templates/etc/e-smith/sql/init/77mysql_update_privs
--- e-smith-horde-1.13.0/root/etc/e-smith/templates/etc/e-smith/sql/init/77mysql_update_privs	2007-01-20 14:57:44.000000000 -0700
+++ mezzanine_patched_e-smith-horde-1.13.0/root/etc/e-smith/templates/etc/e-smith/sql/init/77mysql_update_privs	1969-12-31 17:00:00.000000000 -0700
@@ -1,3 +0,0 @@
-#! /bin/sh
-
-exec mysql < /home/httpd/html/horde/scripts/db/mysql_update_privs.sql
diff -Nur -x '*.orig' -x '*.rej' e-smith-horde-1.13.0/root/etc/e-smith/templates.metadata/etc/e-smith/sql/init/77horde_mysql_update_privs mezzanine_patched_e-smith-horde-1.13.0/root/etc/e-smith/templates.metadata/etc/e-smith/sql/init/77horde_mysql_update_privs
--- e-smith-horde-1.13.0/root/etc/e-smith/templates.metadata/etc/e-smith/sql/init/77horde_mysql_update_privs	1969-12-31 17:00:00.000000000 -0700
+++ mezzanine_patched_e-smith-horde-1.13.0/root/etc/e-smith/templates.metadata/etc/e-smith/sql/init/77horde_mysql_update_privs	2007-01-20 14:55:30.000000000 -0700
@@ -0,0 +1 @@
+PERMS=0540
diff -Nur -x '*.orig' -x '*.rej' e-smith-horde-1.13.0/root/etc/e-smith/templates.metadata/etc/e-smith/sql/init/77mysql_update_privs mezzanine_patched_e-smith-horde-1.13.0/root/etc/e-smith/templates.metadata/etc/e-smith/sql/init/77mysql_update_privs
--- e-smith-horde-1.13.0/root/etc/e-smith/templates.metadata/etc/e-smith/sql/init/77mysql_update_privs	2007-01-20 14:57:44.000000000 -0700
+++ mezzanine_patched_e-smith-horde-1.13.0/root/etc/e-smith/templates.metadata/etc/e-smith/sql/init/77mysql_update_privs	1969-12-31 17:00:00.000000000 -0700
@@ -1 +0,0 @@
-PERMS=0540
