diff -Nur -x '*.orig' -x '*.rej' e-smith-horde-1.11.1/root/etc/e-smith/db/configuration/migrate/HordePassword mezzanine_patched_e-smith-horde-1.11.1/root/etc/e-smith/db/configuration/migrate/HordePassword
--- e-smith-horde-1.11.1/root/etc/e-smith/db/configuration/migrate/HordePassword	2005-12-14 16:06:25.755842603 +1100
+++ mezzanine_patched_e-smith-horde-1.11.1/root/etc/e-smith/db/configuration/migrate/HordePassword	2005-12-14 16:06:11.303357173 +1100
@@ -2,7 +2,7 @@
     my $horderec = $DB->get('horde')
 	|| $DB->new_record('horde', {type => 'service'});
     my $horde_pw = $horderec->prop('DbPassword');
-    if (not $horde_pw or length($horde_pw) < 8)
+    if (not $horde_pw or length($horde_pw) < 57)
     {
 	use MIME::Base64 qw(encode_base64);
 
