diff -ruN e-smith-horde-1.11.1-14/root/etc/e-smith/templates/home/httpd/html/horde/config/mime_drivers.php/260msexcel e-smith-horde-1.11.1-15/root/etc/e-smith/templates/home/httpd/html/horde/config/mime_drivers.php/260msexcel
--- e-smith-horde-1.11.1-14/root/etc/e-smith/templates/home/httpd/html/horde/config/mime_drivers.php/260msexcel	2005-08-25 18:02:17.813587154 -0700
+++ e-smith-horde-1.11.1-15/root/etc/e-smith/templates/home/httpd/html/horde/config/mime_drivers.php/260msexcel	2005-08-25 18:05:32.000000000 -0700
@@ -4,7 +4,7 @@
  * This driver requires xlhtml to be installed.
  * xlhtml homepage: http://chicago.sourceforge.net/xlhtml/
  */
-$mime_drivers['horde']['msexcel']['location'] = '/usr/local/bin/xlhtml';
+$mime_drivers['horde']['msexcel']['location'] = '/usr/bin/xlhtml';
 $mime_drivers['horde']['msexcel']['inline'] = false;
 $mime_drivers['horde']['msexcel']['handles'] = array(
     'application/vnd.ms-excel',
diff -ruN e-smith-horde-1.11.1-14/root/etc/e-smith/templates/home/httpd/html/horde/config/mime_drivers.php/270mspowerpoint e-smith-horde-1.11.1-15/root/etc/e-smith/templates/home/httpd/html/horde/config/mime_drivers.php/270mspowerpoint
--- e-smith-horde-1.11.1-14/root/etc/e-smith/templates/home/httpd/html/horde/config/mime_drivers.php/270mspowerpoint	2005-08-25 18:02:17.814586601 -0700
+++ e-smith-horde-1.11.1-15/root/etc/e-smith/templates/home/httpd/html/horde/config/mime_drivers.php/270mspowerpoint	2005-08-25 18:05:46.000000000 -0700
@@ -4,7 +4,7 @@
  * This driver requires ppthtml, included with xlhtml, to be installed.
  * xlhtml homepage: http://chicago.sourceforge.net/xlhtml/
  */
-$mime_drivers['horde']['mspowerpoint']['location'] = '/usr/local/bin/ppthtml';
+$mime_drivers['horde']['mspowerpoint']['location'] = '/usr/bin/ppthtml';
 $mime_drivers['horde']['mspowerpoint']['inline'] = false;
 $mime_drivers['horde']['mspowerpoint']['handles'] = array(
     'application/vnd.ms-powerpoint',
