diff -Nur -x '*.orig' -x '*.rej' e-smith-horde-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/conf.php/00header mezzanine_patched_e-smith-horde-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/conf.php/00header
--- e-smith-horde-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/conf.php/00header	2007-03-25 18:10:57.000000000 -0500
+++ mezzanine_patched_e-smith-horde-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/conf.php/00header	2007-03-25 16:33:33.000000000 -0500
@@ -1,2 +1,2 @@
 /* CONFIG START. DO NOT CHANGE ANYTHING IN OR AFTER THIS LINE. */
-// $Horde: horde/config/conf.xml,v 1.74.2.41 2006/08/04 17:13:25 jan Exp $
+// $Horde: horde/config/conf.xml,v 1.74.2.48 2007/02/02 11:03:05 jan Exp $
diff -Nur -x '*.orig' -x '*.rej' e-smith-horde-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/conf.php/100GeneralSettings mezzanine_patched_e-smith-horde-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/conf.php/100GeneralSettings
--- e-smith-horde-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/conf.php/100GeneralSettings	2006-03-15 17:51:23.000000000 -0600
+++ mezzanine_patched_e-smith-horde-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/conf.php/100GeneralSettings	2007-03-25 16:37:12.000000000 -0500
@@ -1,11 +1,11 @@
 // 100GeneralSettings
 $conf['debug_level'] = E_ALL;
 $conf['max_exec_time'] = 0;
+$conf['compress_pages'] = true;
+$conf['umask'] = 077;
 $conf['use_ssl'] = 2;
 $conf['server']['name'] = $_SERVER['SERVER_NAME'];
 $conf['server']['port'] = $_SERVER['SERVER_PORT'];
-$conf['compress_pages'] = true;
-$conf['umask'] = 077;
 $conf['session']['name'] = 'Horde';
 $conf['session']['cache_limiter'] = 'nocache';
 $conf['session']['timeout'] = 0;
diff -Nur -x '*.orig' -x '*.rej' e-smith-horde-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/conf.php/110SQL mezzanine_patched_e-smith-horde-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/conf.php/110SQL
--- e-smith-horde-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/conf.php/110SQL	2006-03-15 17:51:24.000000000 -0600
+++ mezzanine_patched_e-smith-horde-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/conf.php/110SQL	2007-03-25 16:42:16.000000000 -0500
@@ -1,6 +1,5 @@
 // 110SQL
 $conf['sql']['persistent'] = false;
-$conf['sql']['hostspec'] = 'localhost';
 $conf['sql']['username'] = 'horde';
 {
     $horderec = $DB->get('horde');
diff -Nur -x '*.orig' -x '*.rej' e-smith-horde-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/conf.php/320Hooks mezzanine_patched_e-smith-horde-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/conf.php/320Hooks
--- e-smith-horde-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/conf.php/320Hooks	2006-03-15 17:51:23.000000000 -0600
+++ mezzanine_patched_e-smith-horde-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/conf.php/320Hooks	2007-03-25 16:57:27.000000000 -0500
@@ -1,4 +1,5 @@
 // 320Hooks
+$conf['hooks']['permsdenied'] = false;
 $conf['hooks']['username'] = false;
 $conf['hooks']['preauthenticate'] = false;
 $conf['hooks']['postauthenticate'] = false;
diff -Nur -x '*.orig' -x '*.rej' e-smith-horde-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/prefs.php/00header mezzanine_patched_e-smith-horde-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/prefs.php/00header
--- e-smith-horde-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/prefs.php/00header	2007-03-25 18:10:57.000000000 -0500
+++ mezzanine_patched_e-smith-horde-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/prefs.php/00header	2007-03-25 17:50:42.000000000 -0500
@@ -1,6 +1,6 @@
 //00header
 /**
- * $Horde: horde/config/prefs.php.dist,v 1.85.2.7 2006/06/22 18:47:55 chuck Exp $
+ * $Horde: horde/config/prefs.php.dist,v 1.85.2.8 2006/12/08 15:29:16 jan Exp $
  *
  * Preferences Information
  * =======================
@@ -27,34 +27,47 @@
  * The $_prefs array's are listed in the same order as listed in the
  * members element of $prefGroups.
  *
- *   value: This entry will either hold a number or a text value based on the
+ *   value: This entry will hold the default preference value depending on the
  *          preference type:
+ *            checkbox:  0 or false for unchecked, 1 or true for checked
+ *                enum:  Preselected item from associated enumeration
  *            implicit:  See Preference type
- *                text:  Text value
+ *                link:  Not used
  *              number:  Number value
- *            checkbox:  Value should be 0 for unchecked, 1 for checked
- *              select:  Value associated with that selection list
- *
+ *            password:  Should be ''
+ *              select:  Preselected item from associated selection list
+ *             special:  Not used
+ *                text:  Text value
+ *            textarea:  Text value, lines separated with "\n"
+ * 
  *  locked: Allow preference to be changed from UI
- *             true:  Do not show this preference in the UI
- *            false:  Show this preference in the UI and allow changing
- *
+ *             true:  Do not show this preference in the UI.
+ *                    This has no effect on 'link' type preferences.
+ *            false:  Show this preference in the UI and allow changing.
+ * 
  *  shared: Share with other horde apps
  *             true:  Share this pref with other Horde apps
  *            false:  Keep this pref local to the current app
  *
  *    type: Preference type
- *             special:  Provides a UI widget
- *              select:  Provides a selection list in the UI
- *            checkbox:  Provides a checkbox
- *            implicit:  Provides storage for 'special' types
+ *            checkbox:  Provides a checkbox.
+ *                enum:  Provides a selection list in the UI, list is
+ *                       specified in the associated 'enum' setting.
+ *            implicit:  Provides storage for 'special' types.
+ *                link:  Provides a link to another data entry form.
+ *              number:  Provides a 3-character textbox to enter a natural
+ *                       number; syntaxcheck is performed after data entry.
  *            password:  Provides a textbox for password entry.
- *                enum:  Use static list of elements...similar to 'select'
- *
- *    enum: Static list of elements.
- *
- * escaped: For an enum or a select, are the keys and values already html-escaped?
- *          Defaults to false if not present.
+ *              select:  Provides a selection list in the UI that is built in
+ *                       lib/prefs.php.
+ *             special:  Provides an UI widget.
+ *                text:  Provides a single-line textbox.
+ *            textarea:  Provides a multi-line textbox.   
+ *
+ *    enum: Static list of elements for 'type' => 'enum'.
+ * 
+ * escaped: For an enum or a select, are the keys and values already
+ *          html-escaped? Defaults to false if not present.
  *
  *    hook: Call a hook function for the value of this preference
  *             true:  Will call the function _prefs_hook_<prefname>
diff -Nur -x '*.orig' -x '*.rej' e-smith-horde-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/registry.php/00header mezzanine_patched_e-smith-horde-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/registry.php/00header
--- e-smith-horde-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/registry.php/00header	2007-03-25 18:10:57.000000000 -0500
+++ mezzanine_patched_e-smith-horde-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/registry.php/00header	2007-03-25 18:09:20.000000000 -0500
@@ -2,7 +2,7 @@
 /**
  * registry.php -- Horde application registry.
  *
- * $Horde: horde/config/registry.php.dist,v 1.255.2.17 2006/07/21 15:49:46 jan Exp $
+ * $Horde: horde/config/registry.php.dist,v 1.255.2.19 2006/11/24 10:48:30 jan Exp $
  *
  * This configuration file is used by Horde to determine which Horde
  * applications are installed and where, as well as how they interact.
@@ -38,3 +38,8 @@
  * url           string   The (optional) URL of 'heading' entries.
  */
 
+// We try to automatically determine the proper webroot for Horde here. This
+// still assumes that applications live under horde/. If this results in
+// incorrect results for you, simply change the 'webroot' setting in the
+// 'horde' stanza below.
+
diff -Nur -x '*.orig' -x '*.rej' e-smith-horde-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/registry.php/100webroot mezzanine_patched_e-smith-horde-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/registry.php/100webroot
--- e-smith-horde-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/registry.php/100webroot	2006-03-15 17:51:24.000000000 -0600
+++ mezzanine_patched_e-smith-horde-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/registry.php/100webroot	1969-12-31 18:00:00.000000000 -0600
@@ -1,21 +0,0 @@
-// 100webroot
-// We try to automatically determine the proper webroot for Horde  
-// here. This still assumes that applications live under horde/. If
-// this results in incorrect results for you, simply change the two
-// uses of the $webroot variable in the 'horde' stanza below.
-//
-// Note for Windows users: the below assumes that your PHP_SELF
-// variable uses forward slashes. If it does not, you'll have to tweak
-// this.
-if (isset($_SERVER['PHP_SELF'])) \{
-    $webroot = preg_split(';/;', $_SERVER['PHP_SELF'], 2, PREG_SPLIT_NO_EMPTY);
-    $webroot = strstr(dirname(__FILE__), '/' . array_shift($webroot));
-    if ($webroot !== false) \{
-        $webroot = preg_replace(';/config$;', '', $webroot);
-    \} else \{
-        $webroot = '/horde';
-    \}
-\} else \{
-    $webroot = '/horde';
-\}
-
diff -Nur -x '*.orig' -x '*.rej' e-smith-horde-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/registry.php/110AppRegistryHorde mezzanine_patched_e-smith-horde-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/registry.php/110AppRegistryHorde
--- e-smith-horde-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/registry.php/110AppRegistryHorde       2006-03-15 17:51:23.000000000 -0600
+++ mezzanine_patched_e-smith-horde-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/registry.php/110AppRegistryHorde2007-03-29 20:25:25.000000000 -0500
@@ -1,7 +1,7 @@
 // 110AppRegistryHorde
 $this->applications['horde'] = array(
     'fileroot' => dirname(__FILE__) . '/..',
-    'webroot' => $webroot,
+    'webroot' => '/horde',
     'initial_page' => 'login.php',
     'name' => _("Horde"),
     'status' => 'active',
diff -Nur -x '*.orig' -x '*.rej' e-smith-horde-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/registry.php/520AppRegistryMidas mezzanine_patched_e-smith-horde-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/registry.php/520AppRegistryMidas
--- e-smith-horde-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/registry.php/520AppRegistryMidas	2007-03-25 18:10:57.000000000 -0500
+++ mezzanine_patched_e-smith-horde-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/registry.php/520AppRegistryMidas	2007-03-25 17:38:30.000000000 -0500
@@ -4,6 +4,6 @@
     'webroot' => $this->applications['horde']['webroot'] . '/midas',
     'name' => _("Ads"),
     'status' => 'inactive',
-    'menu_parent' => 'website'
+    'menu_parent' => 'office'
 );
 
diff -Nur -x '*.orig' -x '*.rej' e-smith-horde-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/registry.php/800WebRoot mezzanine_patched_e-smith-horde-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/registry.php/800WebRoot
--- e-smith-horde-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/registry.php/800WebRoot	1969-12-31 18:00:00.000000000 -0600
+++ mezzanine_patched_e-smith-horde-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/registry.php/800WebRoot	2007-03-25 17:42:07.000000000 -0500
@@ -0,0 +1,28 @@
+// 800webRoot
+function _detect_webroot()
+\{
+    // Note for Windows users: the below assumes that your PHP_SELF variable
+    // uses forward slashes. If it does not, you'll have to tweak this.
+    if (isset($_SERVER['SCRIPT_URL']) || isset($_SERVER['SCRIPT_NAME'])) \{
+        $path = empty($_SERVER['SCRIPT_URL']) ?
+            $_SERVER['SCRIPT_NAME'] :
+            $_SERVER['SCRIPT_URL'];
+        $hordedir = basename(preg_replace(';/config/registry.php$;', '', __FILE__));
+        $webroot = preg_replace(';/' . $hordedir . '.*;', '/' . $hordedir, $path);
+    \} elseif (isset($_SERVER['PHP_SELF'])) \{
+        $webroot = preg_split(';/;', $_SERVER['PHP_SELF'], 2, PREG_SPLIT_NO_EMPTY);
+        $webroot = strstr(dirname(__FILE__), DIRECTORY_SEPARATOR . array_shift($webroot));
+        if ($webroot !== false) \{
+            $webroot = preg_replace(array('/\\\\/', ';/config$;'), array('/', ''), $webroot);
+        \} elseif ($webroot === false) \{
+            $webroot = '';
+        \} else \{
+            $webroot = '/horde';
+        \}
+    \} else \{
+        $webroot = '/horde';
+    \}
+    
+    return $webroot;
+\}
+
