diff -Nur -x '*.orig' -x '*.rej' e-smith-horde-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/registry.php/00header mezzanine_patched_e-smith-horde-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/registry.php/00header
--- e-smith-horde-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/registry.php/00header	2007-11-15 21:56:29.000000000 -0700
+++ mezzanine_patched_e-smith-horde-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/registry.php/00header	2007-11-15 21:30:13.000000000 -0700
@@ -2,7 +2,7 @@
 /**
  * registry.php -- Horde application registry.
  *
- * $Horde: horde/config/registry.php.dist,v 1.255.2.19 2006/11/24 10:48:30 jan Exp $
+ * $Horde: horde/config/registry.php.dist,v 1.255.2.20 2007/03/29 23:02:48 jan Exp $
  *
  * This configuration file is used by Horde to determine which Horde
  * applications are installed and where, as well as how they interact.
diff -Nur -x '*.orig' -x '*.rej' e-smith-horde-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/registry.php/800WebRoot mezzanine_patched_e-smith-horde-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/registry.php/800WebRoot
--- e-smith-horde-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/registry.php/800WebRoot	2007-11-15 21:56:29.000000000 -0700
+++ mezzanine_patched_e-smith-horde-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/registry.php/800WebRoot	2007-11-15 21:29:41.000000000 -0700
@@ -7,7 +7,8 @@
         $path = empty($_SERVER['SCRIPT_URL']) ?
             $_SERVER['SCRIPT_NAME'] :
             $_SERVER['SCRIPT_URL'];
-        $hordedir = basename(preg_replace(';/config/registry.php$;', '', __FILE__));
+        $hordedir = str_replace(DIRECTORY_SEPARATOR, '/', __FILE__);
+        $hordedir = basename(preg_replace(';/config/registry.php$;', '', $hordedir));
         $webroot = preg_replace(';/' . $hordedir . '.*;', '/' . $hordedir, $path);
     \} elseif (isset($_SERVER['PHP_SELF'])) \{
         $webroot = preg_split(';/;', $_SERVER['PHP_SELF'], 2, PREG_SPLIT_NO_EMPTY);
