diff -Nur -x '*.orig' -x '*.rej' e-smith-horde-1.12.0/root/etc/e-smith/templates/home/httpd/html/horde/config/prefs.php/00header mezzanine_patched_e-smith-horde-1.12.0/root/etc/e-smith/templates/home/httpd/html/horde/config/prefs.php/00header
--- e-smith-horde-1.12.0/root/etc/e-smith/templates/home/httpd/html/horde/config/prefs.php/00header	2006-03-16 10:51:24.000000000 +1100
+++ mezzanine_patched_e-smith-horde-1.12.0/root/etc/e-smith/templates/home/httpd/html/horde/config/prefs.php/00header	2006-04-06 11:43:18.934764997 +1000
@@ -1,6 +1,6 @@
 //00header
 /**
- * $Horde: horde/config/prefs.php.dist,v 1.85.2.3 2005/08/23 17:14:42 mrubinsk Exp $
+ * $Horde: horde/config/prefs.php.dist,v 1.85.2.3.2.1 2005/12/27 17:02:01 chuck Exp $
  *
  * Preferences Information
  * =======================
@@ -27,36 +27,39 @@
  * The $_prefs array's are listed in the same order as listed in the
  * members element of $prefGroups.
  *
- *  value: This entry will either hold a number or a text value based on the
- *         preference type:
- *           implicit:  See Preference type
- *               text:  Text value
- *             number:  Number value
- *           checkbox:  Value should be 0 for unchecked, 1 for checked
- *             select:  Value associated with that selection list
- *
- * locked: Allow preference to be changed from UI
- *            true:  Do not show this preference in the UI
- *           false:  Show this preference in the UI and allow changing
- *
- * shared: Share with other horde apps
- *            true:  Share this pref with other Horde apps
- *           false:  Keep this pref local to IMP
- *
- *   type: Preference type
- *            special:  Provides a UI widget
- *             select:  Provides a selection list in the UI
- *           checkbox:  Provides a checkbox
- *           implicit:  Provides storage for 'special' types
- *           password:  Provides a textbox for password entry.
- *               enum:  Use static list of elements...similar to 'select'
- *
- *   enum: Static list of elements.
- *
- *   hook: Call a hook function for the value of this preference
- *            true:  Will call the function _prefs_hook_<prefname>
- *                   to fill in the value of this preference.
- *                   See hooks.php for more details.
- *           false:  Normal behaviour - no hook is called.
+ *   value: This entry will either hold a number or a text value based on the
+ *          preference type:
+ *            implicit:  See Preference type
+ *                text:  Text value
+ *              number:  Number value
+ *            checkbox:  Value should be 0 for unchecked, 1 for checked
+ *              select:  Value associated with that selection list
+ *
+ *  locked: Allow preference to be changed from UI
+ *             true:  Do not show this preference in the UI
+ *            false:  Show this preference in the UI and allow changing
+ *
+ *  shared: Share with other horde apps
+ *             true:  Share this pref with other Horde apps
+ *            false:  Keep this pref local to the current app
+ *
+ *    type: Preference type
+ *             special:  Provides a UI widget
+ *              select:  Provides a selection list in the UI
+ *            checkbox:  Provides a checkbox
+ *            implicit:  Provides storage for 'special' types
+ *            password:  Provides a textbox for password entry.
+ *                enum:  Use static list of elements...similar to 'select'
+ *
+ *    enum: Static list of elements.
+ *
+ * escaped: For an enum or a select, are the keys and values already html-escaped?
+ *          Defaults to false if not present.
+ *
+ *    hook: Call a hook function for the value of this preference
+ *             true:  Will call the function _prefs_hook_<prefname>
+ *                    to fill in the value of this preference.
+ *                    See hooks.php for more details.
+ *            false:  Normal behaviour - no hook is called.
  */
 
diff -Nur -x '*.orig' -x '*.rej' e-smith-horde-1.12.0/root/etc/e-smith/templates/home/httpd/html/horde/config/prefs.php/200personal mezzanine_patched_e-smith-horde-1.12.0/root/etc/e-smith/templates/home/httpd/html/horde/config/prefs.php/200personal
--- e-smith-horde-1.12.0/root/etc/e-smith/templates/home/httpd/html/horde/config/prefs.php/200personal	2006-03-16 10:51:23.000000000 +1100
+++ mezzanine_patched_e-smith-horde-1.12.0/root/etc/e-smith/templates/home/httpd/html/horde/config/prefs.php/200personal	2006-04-06 11:43:18.935764821 +1000
@@ -99,6 +99,7 @@
     'locked' => false,
     'shared' => true,
     'type' => 'select',
+    'escaped' => true,
     'desc' => _("Select your preferred language:")
 );
 
