diff -Nur -x '*.orig' -x '*.rej' e-smith-horde-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/conf.php/125Authentication mezzanine_patched_e-smith-horde-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/conf.php/125Authentication
--- e-smith-horde-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/conf.php/125Authentication	2006-09-28 22:14:12.000000000 -0500
+++ mezzanine_patched_e-smith-horde-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/conf.php/125Authentication	2006-09-28 21:59:53.000000000 -0500
@@ -3,6 +3,9 @@
 $conf['auth']['checkbrowser'] = true;
 $conf['auth']['alternate_login'] = false;
 $conf['auth']['redirect_on_logout'] = false;
-$conf['auth']['params']['app'] = 'imp';
-$conf['auth']['driver'] = 'application';
+{
+my $Mimp = $mimp{'status'} || 'disabled'; return "\$conf[\'auth\'][\'driver\'] = \'composite\'\;" unless ($Mimp eq "disabled");
+$OUT = '$conf[\'auth\'][\'params\'][\'app\'] = \'imp\';
+$conf[\'auth\'][\'driver\'] = \'application\';'
+}
 
diff -Nur -x '*.orig' -x '*.rej' e-smith-horde-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/conf.php/999mimp mezzanine_patched_e-smith-horde-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/conf.php/999mimp
--- e-smith-horde-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/conf.php/999mimp	1969-12-31 18:00:00.000000000 -0600
+++ mezzanine_patched_e-smith-horde-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/conf.php/999mimp	2006-09-28 22:07:30.000000000 -0500
@@ -0,0 +1,24 @@
+// 999mimp
+{
+my $Mimp = $mimp{'status'} || 'disabled'; return "#Mimp login is currently not enabled.  To enable - config set mimp service status enabled"
+ unless ($Mimp eq "enabled");
+$OUT = '$conf[\'auth\'][\'params\'][\'drivers\'] = array(
+    \'imp\' => array(\'driver\' => \'application\',
+                   \'params\' => array(\'app\' => \'imp\')),
+    \'mimp\' => array(\'driver\' => \'application\',
+                    \'params\' => array(\'app\' => \'mimp\')));
+$conf[\'auth\'][\'params\'][\'loginscreen_switch\'] = \'_horde_select_loginscreen\';
+
+if (!function_exists(\'_horde_select_loginscreen\')) \{
+    function _horde_select_loginscreen()
+    \{
+        require_once \'Horde/Browser.php\';
+        $browser = &new Browser();
+        if ($browser->isMobile()) \{
+            return \'mimp\';
+        \}
+        return \'imp\';
+    \}
+\}'
+}
+
