diff -Nur -x '*.orig' -x '*.rej' e-smith-horde-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/mime_drivers.php/260msexcel mezzanine_patched_e-smith-horde-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/mime_drivers.php/260msexcel
--- e-smith-horde-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/mime_drivers.php/260msexcel	2006-11-25 10:33:04.000000000 -0600
+++ mezzanine_patched_e-smith-horde-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/mime_drivers.php/260msexcel	2006-11-25 10:31:37.000000000 -0600
@@ -4,7 +4,7 @@
  * This driver requires xlhtml to be installed.
  * xlhtml homepage: http://chicago.sourceforge.net/xlhtml/
  */
-$mime_drivers['horde']['msexcel']['location'] = '/usr/local/bin/xlhtml';
+$mime_drivers['horde']['msexcel']['location'] = '/usr/bin/xlhtml';
 $mime_drivers['horde']['msexcel']['inline'] = false;
 $mime_drivers['horde']['msexcel']['handles'] = array(
     'application/vnd.ms-excel',
diff -Nur -x '*.orig' -x '*.rej' e-smith-horde-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/mime_drivers.php/270mspowerpoint mezzanine_patched_e-smith-horde-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/mime_drivers.php/270mspowerpoint
--- e-smith-horde-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/mime_drivers.php/270mspowerpoint	2006-11-25 10:33:04.000000000 -0600
+++ mezzanine_patched_e-smith-horde-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/mime_drivers.php/270mspowerpoint	2006-11-25 10:31:53.000000000 -0600
@@ -4,7 +4,7 @@
  * This driver requires ppthtml, included with xlhtml, to be installed.
  * xlhtml homepage: http://chicago.sourceforge.net/xlhtml/
  */
-$mime_drivers['horde']['mspowerpoint']['location'] = '/usr/local/bin/ppthtml';
+$mime_drivers['horde']['mspowerpoint']['location'] = '/usr/bin/ppthtml';
 $mime_drivers['horde']['mspowerpoint']['inline'] = false;
 $mime_drivers['horde']['mspowerpoint']['handles'] = array(
     'application/vnd.ms-powerpoint',
